packages/attribute-registry-process/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "pagopa-interop-attribute-registry-process",
+  "version": "1.0.0",
+  "description": "PagoPA Interoperability service for attribute registry",
+  "main": "dist",
+  "type": "module",
+  "scripts": {
+    "test": "vitest",
+    "lint": "eslint . --ext .ts,.tsx",
+    "lint:autofix": "eslint . --ext .ts,.tsx --fix",
+    "format:check": "prettier --check src",
+    "format:write": "prettier --write src",
+    "start": "node --watch --no-warnings --loader ts-node/esm -r 'dotenv-flow/config' ./src/index.ts",
+    "build": "tsc && pnpm run copy-sql-files",
+    "copy-sql-files": "echo TODO",
```

Comment:
we should sync on this. 
In our recent branch https://github.com/pagopa/interop-be-monorepo/pull/81 and https://github.com/pagopa/interop-be-monorepo/pull/82 we're moving the `eventRepository` into the `common` package so there's no need to handle the `sql` files inside a `process` package